client/wildcard/src/components/Icon/Icon.tsx

```diff
@@ -9,26 +9,35 @@ import { ICON_SIZES } from './constants'
 
 import styles from './Icon.module.scss'
 
-export interface IconProps extends Omit<MdiReactIconProps, 'children'> {
+interface BaseIconProps extends Omit<MdiReactIconProps, 'children'> {
     className?: string
     /**
      * The variant style of the icon. defaults to 'sm'
      */
     size?: typeof ICON_SIZES[number]
 }
 
+interface ScreenReaderIconProps extends BaseIconProps {
+    title?: string
+}
+
+interface HiddenIconProps extends BaseIconProps {
+    'aria-hidden'?: true | 'true'
+}
```

Comment:
```suggestion
interface ScreenReaderIconProps extends BaseIconProps {
    title: string
}

interface HiddenIconProps extends BaseIconProps {
    'aria-hidden': true | 'true'
}
```

Once we have finished making our icons accessible, we will need to make this change to enforce it for any future usage